src/main/java/name/nkid00/u00a7/Options.java

```diff
@@ -15,7 +15,10 @@
 public class Options {
     public static File file;
 
+    // Allowing Players to type formatting codes on many occassions.
     public boolean allowFormattingCodes = true;
+    // Disable Sign Update, which will remove formatting codes when confirming input.
```

Comment:
What do you mean by `Disable Sign Update`?

Reply:
disable removing formatting codes when updating sign... sorry